common/src/device.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Debug)]
+pub struct Device {
```

Comment:
why not handle this internally in database api and spit out client contact and device structs?

Reply:
and also please shift the ids and other stuff to be u32 when it comes out of the db